Alignment/OfflineValidation/python/TkAlAllInOneTool/helperFunctions.py

```diff
@@ -207,36 +206,29 @@ def conddb(*args):
     getcommandoutput2(conddb ...) doesn't work, it imports
     the wrong sqlalchemy in CondCore/Utilities/python/conddblib.py
     """
-    global conddbcode
-    from tempfile import mkdtemp, NamedTemporaryFile
+    from tempfile import NamedTemporaryFile
 
-    if conddbcode is None:
-        conddbfile = getCommandOutput2("which conddb").strip()
-        tmpdir = mkdtemp()
-        getCommandOutput2("2to3 -f print -o " + tmpdir + " -n -w " + conddbfile)
-
-        with open(os.path.join(tmpdir, "conddb")) as f:
-            conddb = f.read()
-
-        conddbcode = conddb.replace("sys.exit", "sysexit")
+    with open(getCommandOutput2("which conddb").strip()) as f:
+        conddb = f.read()
 
     def sysexit(number):
         if number != 0:
             raise AllInOneError("conddb exited with status {}".format(number))
     namespace = {"sysexit": sysexit, "conddboutput": ""}
 
+    conddb = conddb.replace("sys.exit", "sysexit")
+
     bkpargv = sys.argv
     sys.argv[1:] = args
     bkpstdout = sys.stdout
-    try:
-        with NamedTemporaryFile(bufsize=0) as sys.stdout:
-            exec(conddbcode, namespace)
-            namespace["main"]()
-            with open(sys.stdout.name) as f:
-                result = f.read()
-    finally:
-        sys.argv[:] = bkpargv
-        sys.stdout = bkpstdout
+    with NamedTemporaryFile(bufsize=0) as sys.stdout:
```

Comment:
Apparently this change introduced a unit test failure in python3 IB, see https://github.com/cms-sw/cmssw/issues/29337.